modules/programs/autorandr.nix

```diff
@@ -55,6 +55,13 @@ let
         default = true;
       };
 
+      crtc = mkOption {
+        type = types.str;
```

Comment:
I would have liked to use `types.nullOr types.ints.positive`since `crtc` is always a positive integer, except when it's set to 0.

Reply:
How about `types.nullOr types.ints.unsigned` then? 

> unsigned integer, meaning >=0

from here:
https://github.com/NixOS/nixpkgs/blob/7c74817881708ad44d9591d74729eb410fff529c/lib/types.nix#L157

Reply:
Oh perfect, I didn't know about this one.